compare.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
You need to add this file to `files` in package.json

Reply:
Great catch, thanks, done 7392ca9